floor/floor/server/server.py

```diff
@@ -210,7 +211,7 @@ def api_brightness():
             abort(400, 'Value must be on range 0.0-1.0')
         controller.set_brightness(brightness)
 
-    return jsonify({'brightness': controller.brightness})
+    return jsonify({'brightness': float(controller.brightness)})
```

Comment:
Does this work for you?  I still seem to be getting `1` back at the browser rather than `1.0`.

Reply:
Works for me, hrm..

Reply:
ok, the raw result has it as `0.0` but the preview tab of chrome dev tools simplifies it to just `0` and from the web interface, its still not showing `100%` when it should. Are you using Chrome?

Reply:
Ok, treating it like a string works for me:
```
-    return jsonify({'brightness': float(controller.brightness)})
+    return jsonify({'brightness': '{:.1f}'.format(controller.brightness)})
```